kemeny_sourcemash.tex

```diff
@@ -43,15 +43,55 @@ \section{Project Overview}
 
 \section{User Instructions}
 
-While 
-
-\subsection{Production: www.sourcemash.com}
-
-When
-
-\subsection{Development: localhost}
-
-When
+Sourcemash is a web application that is hosted online at \url{www.sourcemash.com}. The current development release is available at \url{staging.sourcemash.com}, and the source code for Sourcemash is available at \url{https://github.com/sourcemash/Sourcemash}. Instructions for installation, execution, and testing in the local environment are included in the README and in �2.2 below.
+
+\subsection{Production}
+
+Users are encouraged to use the production release of Sourcemash available at \url{www.sourcemash.com}. The website uses Material Design techniques to create an easy-to-navigate, responsive user experience. Users are encouraged to sign up or log in to take advantages of all of the features that Sourcemash has to offer, and a continuously available help menu is available by pressing the ``?'' button in the bottom right corner of every page.
+
+The navigation menu, pinned to the left side of the page, is the primary tool used to access features and pages on the site, which are delineated below:
+
+\begin{itemize}
+	\item \textbf{Browse} (\url{sourcemash.com/browse}): The Browse page contains an offering of feeds that users can choose to subscribe to. If users do not see a feed that they wish to add, they may add their own using the form at the top of the page.
+	\item \textbf{Categorizer} (\url{sourcemash.com/categorizer}): This page offers try-it-yourself access to the Sourcemash categorization engine. The categorizer allows users to input the URL of any online article and returns auto-generated categories based on computational linguistic analysis.
+	\item \textbf{Category or Feed View} (e.g. \url{sourcemash.com/feeds/1}): By clicking on a Category or Feed title under the tabs in the left-pinned navigation menu, users can view all items present in that category or feed and interact with content by up/down-voting and saving items.
+	\item \textbf{Saved} (\url{sourcemash.com/saved}): Authenticated users can find the list of items that they have bookmarked here.
+	\item \textbf{Profile} (\url{sourcemash.com/profile}): Authenticated users can toggle a ``Suggested Content'' feature here, whereby Sourcemash will display unsubscribed content to a user in order to introduce him or her to recommended, similar content. For example, if users are subscribed to the \emph{New York Times} and are currently viewing a category titled ``Apple Watch,'' they may also be introduced to articles about the Apple Watch from other sources such as TechCrunch, Macrumors, or Bloomberg, if this feature is enabled.
```

Comment:
Authenticated users can manage their settings here. For example, Sourcemash offers a "Show Suggested Content" feature that will display one additional article inside a category page from a feed that a user is not subscribed to. This helps introduce users to new feeds.